sklearn/_build_utils/__init__.py

```diff
@@ -1,116 +1,3 @@
 """
 Utilities useful during the build.
 """
```

Comment:
I guess you can just remove the full `_build_utils` folder at once since there's nothing left besides this docstring

Reply:
never mind there are other files...

Reply:
I removed the docstring but there are stuff that are used e.g. https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/_build_utils/tempita.py https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/_build_utils/version.py. To be honest not sure why they are under `sklearn` since it feels like maybe they don't have to be? I mirrored Numpy and Scipy

Reply:
Yeah you can ignore this comment, I thought you were removing all files and content from this folder, but realized afterward that there are other files left.

Reply:
> Edit: maybe there is a use case when installing from sdist, not sure ...

**Not actionable for this PR**: I can not think of a good reason for having them under `sklearn`. `_build_utils` can be in the `sdist` at the root level.

Reply:
Yep I kind of agree ...